stock_warehouse_calendar/README.rst

```diff
@@ -0,0 +1,80 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :target: https://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+
+========================
+Stock Warehouse Calendar
+========================
+
+This module adds a Calendar to the Warehouse. This calendar can then used as
+the basis of the proper computation of start/end dates based on lead times in
+this and other modules.
+
+In this module, the calendar considered in the computation of start date of
+stock moves and pickings created from procurements, where the lead time
+is used.
+
+
+Configuration
+=============
+
+* Go to *Settings* and activate the developer mode.
+
+* Go to *Settings > Technical > Resource > Working Time* and define your
+  resource calendar.
+
```

Comment:
Add:

`* Go to *Inventory > Configuration > Settings* and in *Warehouse* mark 'Multi-Step Routes option'.`